src/components/TextField/TextField.tsx

```diff
@@ -186,11 +186,13 @@ class TextField extends React.PureComponent<CombinedProps, State> {
       polaris: {intl},
     } = this.props;
 
+    const normalizedValue = value ? value : '';
```

Comment:
Nitpick here, but our [preference for null or undefined checks](https://github.com/Shopify/javascript#11.6) is `value == null` 

```suggestion
    const normalizedValue = value == null ? value : '';
```

Reply:
Wouldn't this mean that `undefined` no longer works to defaulting to the empty string now, which was the behavior before with the default value in the destructuring? Or are you saying we should do a check like `value === null || value === undefined`?

Reply:
Sorry, typo! `value != null`

Reply:
> EDIT: Just noticed the double equals, not triple. Makes sense. :)

Yep, exactly. The "Abstract Equality Comparison" (I think that’s the official ES2015 name) does a strict check for `null` or `undefined`

Reply:
Thanks @tmlayton 👍, I committed that change.